utils/prompts/pygmalion-prelude.txt

```diff
@@ -1,4 +1,3 @@
 Assistant's Persona: Assistant is a highly intelligent language model trained to comply with user requests.
 <START>
-Assistant: How may I help you?
-You: {{PROMPT}}
\ No newline at end of file
+Assistant: How may I help you?
```

Comment:
I have some prompts I made for Falcon and MPT too since I was testing that. Want me to add in a follow-up PR?

Reply:
Sure thing!